gui-tk/godley.tcl

```diff
@@ -230,12 +230,12 @@ proc godleyContext {id x y X Y} {
             $id.insertIdx 0
             $id.selectIdx 0
         }
-        if [string length [$id.godleyIcon.table.getCell $r $c]] {
+        if {[string length [$id.godleyIcon.table.getCell $r $c]] && [$id.godleyIcon.table.getCell $r $c]!=[$id.godleyIcon.table.getCell 1 0]} {   # Cannot Cut or Copy cell(1,0). For ticket 1064
```

Comment:
Why not make the condition && ($r!=1 || $c!=0) ? Currently, you are just testing that the source cell is not "Iniitial  condtitions", which is not the same thing.

Reply:
Also, there is no problem in allowing the user to copy cell (1,0).

Reply:
Thanks :) Yes, testing for string "Initial Conditions" produces unexpected behaviour in other cells.